wagtail/wagtailsearch/tests/test_elasticsearch2_backend.py

```diff
@@ -433,7 +433,7 @@ def test_isnull_false_lookup(self):
         # Check it
         expected_result = {'filtered': {'filter': {'and': [
             {'match': {'content_type': 'searchtests.SearchTest'}},
-            {'not': {'missing': {'field': 'title_filter'}}}
+            {'exists': {'field': 'title_filter'}}
```

Comment:
See: https://github.com/torchbox/wagtail/pull/3145/commits/b8f267aa65e0dfb3a86e1aefbbaa370610dd568b (this one affects all backends)